piq/brisque.py

```diff
@@ -53,7 +53,7 @@ def brisque(x: torch.Tensor,
     assert kernel_size % 2 == 1, f'Kernel size must be odd, got [{kernel_size}]'
     _validate_input([x, ], dim_range=(4, 4), data_range=(0, data_range))
 
-    x = x / data_range * 255
+    x = x / float(data_range) * 255
```

Comment:
Why is this _explicit_ conversion necessary?

Reply:
In case of `x` and `data_range` being `torch.uint8` and `int`, the result will be `Byte` for `torch==1.5.1` due to classic division instead of the `true_division`. It will lead to zeroing the input and possible exceptions from pytorch layers, which could be incompatible with `torch.Byte`.

Reply:
Agh, I see. This is unfortunate because from one hand we don't check user's inputs but from the other we require `data_range` to be `int` of `float`. I think this is a good compromise though.